google/cloud/storage/bucket.py

```diff
@@ -84,6 +84,8 @@
 )
 _API_ACCESS_ENDPOINT = "https://storage.googleapis.com"
 
+_PUBLIC_ACCESS_PREVENTION_UNSPECIFIED = "unspecified"
```

Comment:
Add constant for enforced as well. And use these constants in your system tests rather than redefining there.